package.json

```diff
@@ -3,6 +3,15 @@
   "version": "1.4.1",
   "description": "MagicMirror² Module that displays the current moon phase",
   "main": "MMM-MoonPhase.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/NolanKingdon/MMM-MoonPhase.git"
+  },
+  "keywords": [
+    "MagicMirror",
+    "Moon",
```

Comment:
What do you think about adding "Astronomy"? That would be a nice main category in the new module list.